ruma-identity-service-api/src/tos/get_terms_of_service/v2.rs

```diff
@@ -0,0 +1,62 @@
+//! [GET /_matrix/identity/v2/terms](https://matrix.org/docs/spec/identity_service/r0.3.0#get-matrix-identity-v2-terms)
+
+use std::collections::BTreeMap;
+
+use ruma_api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+        metadata: {
+            description: "Gets all the terms of service offered by the server. The client is expected to filter through the terms to determine which terms need acceptance from the user. Note that this endpoint does not require authentication.",
+            method: GET,
+            name: "tos::get_terms_of_service::v2",
+            path: "/_matrix/identity/v2/terms",
+            authentication: None,
+            rate_limited: false,
+        }
+
+        #[derive(Default)]
+        request: {}
+
+        response: {
+            /// Required. The policies the server offers. Mapped from arbitrary ID
```

Comment:
```suggestion
            /// The policies the server offers.
            ///
            /// Mapped from arbitrary ID [...]
```

The `Required.` annotation is not needed for our docs since in our strongly-typed API, users can't accidentally access things that don't exist, or are forced to handle possibly missing values in case of `Option`.

Also it's generally good to separate the "heading" and body of the docs. (the first line is what will show up in search results, I think)

Reply:
Changes made.